pyproject.toml

```diff
@@ -53,6 +54,8 @@ beautifulsoup4 = { version = "^4", optional = true }
 docx2txt = { version = "^0.8", optional = true }
 PyMuPDF = { version = "^1.22.5", optional = true }
 wikipedia = { version = "^1", optional = true }
+duckduckgo-search = "^4.2.0"
```

Comment:
HI, the reason why the default installation is set here is because after the optional installation, the PR cannot pass the CI and this package is not installed.

Reply:
you need to run poetry lock and install after change the toml file